apps/desktop/src/lib/ai/service.test.ts

```diff
@@ -1,15 +1,15 @@
-import { AnthropicAIClient } from '$lib/ai/anthropicClient';
-import { ButlerAIClient } from '$lib/ai/butlerClient';
-import { OpenAIClient } from '$lib/ai/openAIClient';
-import { SHORT_DEFAULT_BRANCH_TEMPLATE, SHORT_DEFAULT_COMMIT_TEMPLATE } from '$lib/ai/prompts';
-import { AISecretHandle, AIService, GitAIConfigKey, KeyOption, buildDiff } from '$lib/ai/service';
```

Comment:
Can we discuss the use of relative imports on Monday? I'm very much not a fan of them.

Reply:
The only exception I make, and subjectively so, is that relative imports are ok for the same package, i.e. `./file.ts`. When it comes to any other import I prefer absolute. What's your view?

Reply:
I would always prefer to use absolute imports. I find it clearer to see what is getting included in files. Always using the absolute imports has a second advantage of making it easier to find and update all the places where a given file is imported. For example, after a rename or large move with the LSP failed to track (like when moving things to the UI library).